src/Reflection/FunctionReflection.php

```diff
@@ -223,6 +223,19 @@ public function getParameters(): array
 					true
 				);
 			}
+
+			if (
+				$this->reflection->getName() === 'get_defined_functions'
+			    && $this->reflection->getNumberOfRequiredParameters() === 1
+			    && count($this->parameters) === 1
+			) {
```

Comment:
I know the "test the functionality and not the version" policy: to me this is the exact case where we should test the version and not the functionality.
What if in the future the function would change the signature?
The bug is present in a specific set of version, we should test those version only.